projects/packages/search/src/dashboard/components/record-meter/index.jsx

```diff
@@ -20,9 +20,15 @@ import './style.scss';
  * @param {number} props.postCount - Post count
  * @param {object} props.postTypeBreakdown - Post type breakdown (post type => number of posts)
  * @param {number} props.tierMaximumRecords - Max number of records allowed in user's current tier
+ * @param {Date} props.lastIndexedDate - The date on which the site was last indexed
```

Comment:
I did a `typeof( lastIndexedDate )` and it's currently a string, not a Date.

One option would be for the selector to create a new date object using `Date.parse`. 

For the moment, let's just update the JSdoc because we don't actually need to work with or display the date (just check that it exists).

```suggestion
 * @param {string} props.lastIndexedDate - The date on which the site was last indexed in ISO 8601 format
```

Some good background about ISO 8601: https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard

Reply:
ah! I'd changed that on my local version but looks like I hadn't pushed those changes for some reason. thanks!